LDAR_Sim/src/virtual_world/fugitive_emission.py

```diff
@@ -162,7 +162,7 @@ def tag_leak(
 
     def calc_mitigated(self, end_date) -> float:
         # Calculate the comparison date
-        comparison_date = self._start_date + timedelta(days=self._nrd)
+        comparison_date = self._start_date + timedelta(days=int(self._nrd))
```

Comment:
Once again, this should already be an integer

Reply:
It's from an int64 to int.